Source/model/draw/Color.swift

```diff
@@ -60,8 +60,9 @@ open class Color: Fill, Equatable {
         return Color( val: ( ( ( x | y ) | z ) | q ) )
     }
 
-    // GENERATED
+    // GENERATED NOT
     open class func rgba(r: Int, g: Int, b: Int, a: Double) -> Color {
+        let a = min(max(a, 0), 1) 
```

Comment:
The place where opacity fixed looks little strange to me. Why do we fix it there, but not in the `rgbt` method? Why do we correct only `a`, but not `r`, `g`, `b` values?

From my point of view correcting all color values is overengineering and we need to assume that this method should be called with valid arguments. So I suggest to fix parser instead of a model.